app/views/todos/todo.erb

```diff
@@ -0,0 +1,32 @@
+<center><h1> The to do list: </h1>
+
+<table>
+ <% @todo.each do|t| %>
+  <tr>
+    <td>
+      <form action ="complete_todo">
```

Comment:
Where is this form supposed to submit to? There's no controller action with this path.

Also how would one even submit this form? There's no submit button.